.travis/tools-and-emacs

```diff
@@ -0,0 +1,23 @@
+# Adding this stage is a workaround because `COPY --from` does not support
+# variable substitution.
+# See https://github.com/moby/moby/issues/34482
+ARG EMACS_VERSION=25.3
+FROM flycheck/emacs-cask:${EMACS_VERSION} AS emacs-cask
+
+FROM flycheck/all-tools
+
+# We need gnutls for downloading packages from ELPA
+RUN apt-get update >/dev/null && \
+    apt-get install --no-install-recommends -y \
+            gnutls-bin \
+            make \
+            >/dev/null \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
When building Docker images, you want them to be a slim as possible.  By default, it will save everything created in the container filesystem.  Doing an `apt-get update` will fetch the lists, so you remove them once you don't need to install anything anymore.